app/models/patient.rb

```diff
@@ -708,9 +708,17 @@ def latest_transfer
   #     AND
   #  - on the last day of or past their monitoring period
   scope :close_eligible, lambda {
-    exposure_asymptomatic
+    no_recent_activity.or(
```

Comment:
Wouldn't this allow any records in? Even in other workflows and line lists? It should be limited to exposure non reporting for now.